test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -14,6 +15,16 @@ public void setUp()
     }
 
     @Test
+    public void constructorReadsPropertiesFile()
+    {
+      WasapiDownloader myInstance = new WasapiDownloader();
+      assertNotNull(myInstance.settings);
+      assertEquals("incorrect default settings.properties baseurl value", myInstance.settings.getProperty("baseurl"), "http://example.org");
+      assertEquals("incorrect default settings.properties username value", myInstance.settings.getProperty("username"), "user");
+      assertEquals("incorrect default settings.properties password value", myInstance.settings.getProperty("password"), "pass");
+    }
+
```

Comment:
I fussed around with a bunch of additional tests, but they fell into these buckets:
- testing what the Properties class does (presumably it's a well tested interface)
- testing how the Properies.load() method deals with missing files, empty files, etc
- testing a private method, loadPropertiesFile()
- not being able to figure out a quick way to mock that a file doesn't exist, or that an exception is thrown ...

I am open to other opinions on what we should be testing